runtime/jcl/common/jclcinit.c

```diff
@@ -170,28 +175,32 @@ jint computeFullVersionString(J9JavaVM* vm)
 	strcat(fullversion, aotEnabled ? "en" : "dis");
 	strcat(vminfo, "abled, AOT ");
 	strcat(vminfo, aotEnabled ? "en" : "dis");
-	strcat(fullversion, "abled)\nJ9VM - ");
-	strcat(vminfo, "abled)\nJ9VM - ");
+	strcat(fullversion, "abled)\n");
+	strcat(fullversion, OPENJ9_REPO);
+	strcat(vminfo, "abled)\n");
+	strcat(vminfo, OPENJ9_REPO);
 
 #endif
 	vmVersion = J9VM_VERSION_STRING;
 	strcat(fullversion, vmVersion);
 	strcat(vminfo, vmVersion);
 
+#if OPENJ9_BUILD
```

Comment:
Was this intended to include JIT information in some case? I don't see JIT in either output for OPENJ9_BUILD = true or false. Seem this code can be deleted.

Reply:
Updated in latest.